src/frontend/pages/recruitment/ApplicationPage/ApplicationPage.js

```diff
@@ -15,16 +14,20 @@ import {
   setCheckboxesShown,
   oneTrue,
   getItemByName,
+  renameObjectKeys,
 } from '../utils';
 
 import useApplications from '../../../hooks/applications';
 import usePostings from '../../../hooks/postings';
+import useTeams from '../../../hooks/teams';
 
 const ApplicationPage = () => {
   const history = useHistory();
 
   const { applications } = useApplications('FALL-2022'); // TODO: in production, replace with Date.now().
   const { postings } = usePostings();
+  const { teams } = useTeams();
+  renameObjectKeys(teams, 'teamName', 'name');
```

Comment:
To reduce the amount of code needed and to keep things more organized, I'd suggest doing this renaming in the Redux logic instead of the React Component file if it doesn't make things more complicated.